### PR TITLE
Remove some moves that are not moves

### PR DESCRIPTION
**Pull request by dimitry-**
And some other minor fixes

Test: builds

### CODE REVIEW

**Review by beckerhe — APPROVED** [review]
LGTM!

File: src/OrbitQt/orbitmainwindow.cpp [hunk 1]
```diff
@@ -470,7 +470,7 @@ void OrbitMainWindow::SetupCaptureToolbar() {
   // Timer
   toolbar->addWidget(CreateSpacer(toolbar));
   QFontMetrics fm(ui->timerLabel->font());
-  int pixel_width = fm.width("w");
+  int pixel_width = fm.horizontalAdvance("w");
```
Comment (florian-kuebler) [inline src/OrbitQt/orbitmainwindow.cpp:473]: This kind of comes out of nothing. Is this part of "minor bugfixes"?

**Review by florian-kuebler — APPROVED** [review]
LGTM

File: src/OrbitQt/orbitmainwindow.cpp (hunk 1 above)
Comment (dimitry-) [reply src/OrbitQt/orbitmainwindow.cpp:473]: width is deprecated so I replaced it. I do not think this fixes anything. just code cleanup.

**Pull request merged**